ucsmsdk/ucssession.py

```diff
@@ -118,6 +121,40 @@ def evt_channel(self):
     def last_update_time(self):
         return self.__last_update_time
 
+    def _freeze(self):
+        save = {
+            "ip": self.__ip,
+            "username": self.__username,
+            "password": self.__password,
+            "proxy": self.__proxy,
+            "uri": self.__uri,
+            "ucs": self.__ucs,
+            "name": self.__name,
+            "cookie": self.__cookie,
+            "session_id": self.__session_id,
+            "version": self.__version,
+            "refresh_period": self.__refresh_period,
+            "priv": self.__priv,
+            "domains": self.__domains,
+            "channel": self.__channel,
+            "evt_channel": self.__evt_channel,
+            "last_update_time": self.__last_update_time,
+            "force": self.__force,
+            "auto_refresh": self.__auto_refresh,
+            "dump_xml": self.__dump_xml,
+            "redirect": self.__redirect
+        }
+        return json.dumps(save)
+
+    def _unfreeze(self, params_json):
```

Comment:
We are not saving `self.__driver`, so if the user does any operation like tech-support download which calls `file_download`, it will fail, since it implicitly picks up `self.__driver`
Should we catch this ?